test/config-test.ts

```diff
@@ -4,7 +4,7 @@ import {normalizeConfig as config, mergeToc, readConfig, setCurrentDate} from ".
 import {LoaderResolver} from "../src/dataloader.js";
 
 describe("readConfig(undefined, root)", () => {
-  before(() => setCurrentDate(new Date("2024-01-11T01:02:03")));
+  before(() => setCurrentDate(new Date("2024-01-10T16:00:00")));
```

Comment:
This change is to match the build tests — since the config is now cached, we want the footer to have the same data, so the behavior of the tests is the same independent of order.